pod/perlfunc.pod

```diff
@@ -10580,6 +10580,8 @@ L<C<$SIG{CHLD}>|perlvar/%SIG>, it may accidentally wait for the child
 created by L<C<qx>|/qxE<sol>STRINGE<sol>> or L<C<system>|/system LIST>.
 See L<perlipc> for details.
 
+Equivalent to L<C<waitpid(-1, 0)>|/waitpid>.
```

Comment:
Isn't that a broken link? I would've expected `L<...|/waitpid PID,FLAGS>` to match the `=item` directive following below.

Reply:
I agree with @mauke.  Some crude experimentation suggests that:
```
diff --git a/../perl/pod/perlfunc.pod b/./pod/perlfuncx.pod
index 3364c1fd5e..fbeb88c462 100644
--- a/../perl/pod/perlfunc.pod
+++ b/./pod/perlfuncx.pod
@@ -10580,6 +10580,8 @@ L<C<$SIG{CHLD}>|perlvar/%SIG>, it may accidentally wait for the child
 created by L<C<qx>|/qxE<sol>STRINGE<sol>> or L<C<system>|/system LIST>.
 See L<perlipc> for details.
 
+Equivalent to L<C<waitpid(-1, 0)>|/waitpid PID,FLAGS>.
+
 Portability issues: L<perlport/wait>.
 
 =item waitpid PID,FLAGS
```
... will produce a correct link to the following `waitpid` entry.

Reply:
I was expecting podcheck to pick up a bad link, but I guess not here.

Reply:
It's probably because waitpid is also a kernel call, and there are these lines in `t/porting/known_pod_issues.dat`

> waitpid(2)
waitpid(3)

Reply:
podcheck deliberately allows such links, if I disallow them there are 39 other similar links, an example is a link to perldoc/recv from IO::Socket.

Both perldoc.perl.org and metacpan.org resolve that (though metacpan [scrolls away a little](https://github.com/metacpan/metacpan-web/issues/2972).

I'll fix this link.